pkgs/data/fonts/ubuntu-font-family/default.nix

```diff
@@ -1,26 +1,30 @@
-{ lib, fetchzip }:
+{ lib, stdenv, fetchzip }:
 
-fetchzip {
-  name = "ubuntu-font-family-0.83";
+stdenv.mkDerivation rec {
+  pname = "ubuntu-font-family";
+  version = "0.83";
 
-  url = "https://assets.ubuntu.com/v1/fad7939b-ubuntu-font-family-0.83.zip";
+  src = fetchzip {
+    url = "https://assets.ubuntu.com/v1/fad7939b-${pname}-${version}.zip";
+    hash = "sha256-FAg1xn8Gcbwmuvqtg9SquSet4oTT9nqE+Izeq7ZMVcA=";
+  };
 
-  postFetch = ''
-    mkdir -p $out/share/fonts
-    unzip -j $downloadedFile \*.ttf -d $out/share/fonts/ubuntu
+  installPhase = ''
+    install -D -m 644 -t "$out/share/fonts/truetype" *.ttf
   '';
 
-  sha256 = "090y665h4kf2bi623532l6wiwkwnpd0xds0jr7560xwfwys1hiqh";
+  outputHashMode = "recursive";
+  outputHash = "sha256-EEcYtOeOd2DKyRLo1kG7lk8euaFilCFMXMJNAosxHiQ=";
```

Comment:
why is this a fixed output derivation?

Reply:
From what I understand, the output only depends on the current version used, so it would make sense to use a fixed-output derivation. Are you seeing something that would make this derivation variable (other than current version)?

Reply:
Probably not, that's just normally not how we use FODs. They're used sparingly only when needed, because they disable part of the sandbox (mainly being able to reach the network).

Reply:
This will break if the build is not reproducible or does not yield the exact same output on all platforms

Reply:
While we're at it, does this **really** need a full C build environment?